### PR TITLE
Added ability to translate some strings("Drives" in navigation, file …

### PR DESCRIPTION
**Pull request by tsvietOK**
…browser columns headers, file types, permission dialog, rename dialog, theme and date style selectors in settings) 
Translation to Russian 
Added Ukrainian translation 
Fixed a bug when file type look changes after reloading page 
Fixed a bug when Russian translation not regenerates as it was included not as xliff resource 
Include Turkish xliff translation file to build

### CODE REVIEW

**Comment by xPoppyx** [comment]
Why does "Text Document" in Spanish need a translation revision?

**Comment by tsvietOK** [comment]
> Why does "Text Document" in Spanish need a translation revision?

Good comment. I will fix that Thx

**Comment by xPoppyx** [comment]
you can add "This folder is empty." ?, I did not find it in the file to be able to translate it

**Comment by tsvietOK** [comment]
> you can add "This folder is empty." ?, I did not find it in the file to be able to translate it

I will try

**Comment by duke7553** [comment]
Thanks a lot guys!

When this is ready to be reviewed, please let me know!

**Comment by tsvietOK** [comment]
@duke7553 If there is no suggestions about translation, let's review it.

File: Files/Filesystem/Drives.cs [hunk 1]
```diff
@@ -58,9 +59,9 @@ private async void DeviceWatcher_EnumerationCompleted(DeviceWatcher sender, obje
 			{
 				await CoreApplication.MainView.Dispatcher.RunAsync(CoreDispatcherPriority.Low, () =>
 				{
-					if (App.sideBarItems.FirstOrDefault(x => x is HeaderTextItem && x.Text == "Drives") == null)
+					if (App.sideBarItems.FirstOrDefault(x => x is HeaderTextItem && x.Text == ResourceController.GetTranslation("Drives")) == null)
 					{
-						App.sideBarItems.Add(new HeaderTextItem() { Text = "Drives" });
+						App.sideBarItems.Add(new HeaderTextItem() { Text = ResourceController.GetTranslation("Drives") });
```
Comment (yaichenbaum) [inline Files/Filesystem/Drives.cs:64]: Can you rename `Drives` to `SidebarDrives` in order to match the rest of the sidebar items?

**Review by yair100 — CHANGES_REQUESTED** [review]

File: Files/Filesystem/Drives.cs (hunk 1 above)
Comment (tsvietOK) [reply Files/Filesystem/Drives.cs:64]: Yeah, i will rename it as soon as possible.

**Comment by yaichenbaum** [comment]
@tsvietOK It looks like there is a merge conflict, other then that, everything looks good.

**Comment by lukeblevins** [comment]
@yair100 Does this look ready to merge now?

**Review by yair100 — APPROVED** [review]

**Comment by yaichenbaum** [comment]
@duke7553 Everything looks good to me.

**Comment by yaichenbaum** [comment]
@tsvietOK Can you do me a favor, and merge with the latest changes in `develop`? After doing that, I should be able to merge your pull request.

**Comment by xPoppyx** [comment]
have these texts been added to the xlf file?

Anotación 2020-04-12 163255

**Comment by yaichenbaum** [comment]
@xPoppyx They have not.

**Comment by yaichenbaum** [comment]
@tsvietOK Despite the pipeline building successfully, I am not able to build the solution locally, can you check if it is working on your end?

**Comment by tsvietOK** [comment]
> @tsvietOK Despite the pipeline building successfully, I am not able to build the solution locally, can you check if it is working on your end?

From my side it's ok

image
Try to rebuild project

**Comment by tsvietOK** [comment]
@yair100 Hmm, there is error https://github.com/duke7553/files-uwp/commit/60147862cb5c9ef780fb6b25924b957dd10042c4
There is missing closing tag trans-unit

**Comment by yaichenbaum** [comment]
@tsvietOK Thank you for catching that. It should be fixed with #584.

**Pull request merged**